samples/Azure.ResourceManager.Sample/Generated/Extensions/SubscriptionExtensions.cs

```diff
@@ -32,8 +31,8 @@ private static SubscriptionExtensionClient GetExtensionClient(Subscription subsc
         /// <param name="subscription"> The <see cref="Subscription" /> instance the method will execute against. </param>
         /// <param name="location"> The name of a supported Azure region. </param>
         /// <param name="publisherName"> The String to use. </param>
-        /// <exception cref="ArgumentException"> <paramref name="location"/> or <paramref name="publisherName"/> is empty. </exception>
-        /// <exception cref="ArgumentNullException"> <paramref name="location"/> or <paramref name="publisherName"/> is null. </exception>
+        /// <exception cref="System.ArgumentException"> <paramref name="location"/> or <paramref name="publisherName"/> is an empty string, and was expected to be non-empty. </exception>
```

Comment:
Can we explicitly add System namespace when we enter the function that is writing these exception comments ?